pkg/controllers/termination/terminate.go

```diff
@@ -83,14 +82,11 @@ func (t *Terminator) drain(ctx context.Context, node *v1.Node) (bool, error) {
 	var podsToEvict []*v1.Pod
 	// Skip node due to pods that are not able to be evicted
 	for _, p := range pods {
-		// if a pod doesn't have owner references then we can't expect a controller to manage its lifecycle
-		if len(p.ObjectMeta.OwnerReferences) == 0 {
-			return false, NodeDrainErr(fmt.Errorf("pod %s/%s does not have any owner references", p.Namespace, p.Name))
-		} else if podutil.HasDoNotEvict(p) {
+		if podutil.HasDoNotEvict(p) {
 			return false, NodeDrainErr(fmt.Errorf("pod %s/%s has do-not-evict annotation", p.Namespace, p.Name))
 		}
 		// Ignore if unschedulable is tolerated, since they will reschedule
-		if (scheduling.Taints{{Key: v1.TaintNodeUnschedulable, Effect: v1.TaintEffectNoSchedule}}).Tolerates(p) == nil {
+		if podutil.ToleratesUnschedulableTaint(p) {
```

Comment:
This feels unnecessary to create a util for this since there is already a `Tolerates` method on a Taint. 

Maybe a `var` statement at the top of the file that lists the unschedulable taint like this

```go
var unschedulableTaint = scheduling.Taints{{Key: v1.TaintNodeUnschedulable, Effect: v1.TaintEffectNoSchedule}}
```

and line 89 can say

```go
unschedulableTaint.Tolerates(p) == nil
```

Reply:
I suppose we could do that, but in my opinion `ToleratesUnschedulableTaint` is more readable.

Reply:
Another option, put `UnschedulableTaint` as a var in the package `scheduling`, and then put `scheduling.UnschedulableTaint.Tolerates(p)` here. All options seem comparable.